Packs/ApiModules/Scripts/AWSApiModule/AWSApiModule.py

```diff
@@ -1,5 +1,6 @@
-from CommonServerPython import *
```

Comment:
I don't see any significant change in this file
Please restore it to the previous state